### ISSUE TITLE
What's the advantage of akka-ui over scalajs-react?

### ISSUE THREAD

**Issue by elyphas** [pishen/akka-ui#1]
Someone could tell me, What the advantages of akka-ui over scalajs-react (reactjs)? Please
I am using scalajs-react but only basically, so it seems it is too much for my projects.
Thanks in advance

**Comment by pishen** [comment]
I think this can be reduced to the comparison between FRP (Functional Reactive Programming) and React? Since I'm not familiar with React, maybe you can check some other discussions like [this](https://www.sitepoint.com/switching-from-react-to-cycle-js/), [this](https://github.com/funkia/turbine/issues/32), or [this](https://stackoverflow.com/questions/1028250/what-is-functional-reactive-programming/1030631#1030631).

AkkaUI is just yet another FRP tool which is built on top of Akka Stream since Akka Stream also supports Reactive Streams. It may be easier to understand for (backend?) people who already know Akka Stream. But it's more like a proof of concept library from me and I've only used it in some small projects. There may be bugs and limited maintenance.

You can also give a check to [OutWatch](https://outwatch.github.io/?lang=scala) and [Laminar](https://github.com/raquo/Laminar) if you want to see more choices. If you like AkkaUI, discussions and contributions are also welcomed. :)

**Comment by elyphas** [comment]
I wish to contribute, 
I am a little newbie in scala but I have made some advances,
I am using in this moment akka with fsm and actors, and mostly
in scala.js and akka.js no much on jvm, I am going to do all suggestion you give me, 
this project seems to me attractive because of akka stream.

Thank You very much

**Comment by elyphas** [comment]
Would You recommend me an router?
I am reading about these:

https://github.com/Voltir/route.rx
https://github.com/werk/router4s

What do You think?

To start with akkaUI on a project alone?

Thank You in advance!

**Comment by pishen** [comment]
I wrote my router using AkkaUI directly. Didn't know theses tools. Would like to give a look.

**Comment by elyphas** [comment]
Yes, I would like to, I search for one but I dind't find any in your repository.

Thank you
